src/util-device.h

```diff
@@ -32,14 +33,14 @@ typedef struct LiveDevice_ {
 } LiveDevice;
 
 
-int LiveRegisterDevice(char *dev);
-int LiveGetDeviceCount(void);
-char *LiveGetDeviceName(int number);
-LiveDevice *LiveGetDevice(char *dev);
-int LiveBuildDeviceList(char * base);
+int LiveRegisterDevice(char *dev, int runmode);
```

Comment:
Instead of 'int runmode' I would like this to take the runmodes enum. Thats currently anonymous, so that will have to change as well.